fluid/faster_rcnn/models/model_builder.py

```diff
@@ -196,12 +196,21 @@ def fast_rcnn_heads(self, roi_input):
             pool_rois = self.rois
         else:
             pool_rois = self.rpn_rois
-        pool = fluid.layers.roi_pool(
-            input=roi_input,
-            rois=pool_rois,
-            pooled_height=14,
-            pooled_width=14,
-            spatial_scale=0.0625)
+        if cfg.roi_func == 'RoIPool':
+            pool = fluid.layers.roi_pool(
+                input=roi_input,
+                rois=pool_rois,
+                pooled_height=14,
+                pooled_width=14,
+                spatial_scale=0.0625)
```

Comment:
pooled_width and spatial_scale should leave as a config in config.py
For example, pooled_height may take 7 or 14

Reply:
Configs are added